manifests/n/NelsonNumericalSoftware/Nelson/0.5.7/NelsonNumericalSoftware.Nelson.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: NelsonNumericalSoftware.Nelson
+PackageVersion: 0.5.7
+PackageLocale: en-US
+Publisher: Nelson Numerical Software
```

Comment:
```suggestion
Publisher: Allan CORNET
```

Reply:
not as person but as organization

Reply:
You'll have to update that in your software then.